test/manager/bundler/artifacts.spec.ts

```diff
@@ -32,16 +34,21 @@ describe('bundler.updateArtifacts()', () => {
     jest.resetModules();
 
     config = {
-      localDir: '/tmp/github/some/repo',
+      // `join` fixes Windows CI
+      localDir: join('/tmp/github/some/repo'),
     };
 
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
+    setDockerConfig({
+      ...config,
+      dockerUser: 'foobar',
+    });
   });
   it('returns null by default', async () => {
     expect(
       await updateArtifacts({
         packageFileName: '',
-        updatedDeps: [],
+        updatedDeps: ['foo', 'bar'],
```

Comment:
Do we have an test for no artifacts? Then we should now call `bundle lock` i think

Reply:
Are empty arrays possible?

Reply:
🤔 @rarkins do you have an answer?

Reply:
They're not meant to be empty